players.sql

```diff
@@ -0,0 +1,32 @@
+ CREATE TABLE players(
+   id serial UNIQUE NOT NULL,
```

Comment:
Nice! You don't have to add the `UNIQUE` and `NOT NULL` constraints for serial. Serial will set those by default.
```suggestion
   id serial,
```